src/formatDate.js

```diff
@@ -50,7 +50,30 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldFormat = date.split(fromFormat[3]);
+  const newFormat = [];
+
+  for (let i = 0; i < 3; i++) {
+    for (let k = 0; k < 3; k++) {
+      if (fromFormat[i] === 'YYYY' && toFormat[k] === 'YY') {
+        newFormat[k] = oldFormat[i].slice(2);
+      }
+
+      if (fromFormat[i] === 'YY' && toFormat[k] === 'YYYY') {
+        if (oldFormat[i] < 30) {
```

Comment:
It is also better to avoid nesting if...else, try to use ternary operator in this case
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

Reply:
thanks for advice)